### PR TITLE
Add repro for #13332

### PR DESCRIPTION
**Pull request by nemanjaglumac**
### Status
READY

### What does this PR accomplish?
- reproduces #13332 

### How to test this manually?
- `yarn test-cypress-open`
- `frontend/test/metabase/scenarios/question/native.cy.spec.js`
- replace `it.skip()` with `it.only()`
- test should fail until the issue is resolved

### Related Issues
- #3354 
- #13158

### Additional notes:
- I started using `[ci skip]` until code reviews are done and PR is absolutely ready for merge (cc @rlotun )
- this is probably a stylistic preference, but I never know should I include test within `forEach()` loop (if we are testing different versions of the same question), or should test include the loop. If anyone has strong preference towards either, please let me know.

### CODE REVIEW

**Review by flamber — COMMENTED** [review]
Not sure why you're creating the question twice. You can use the same source question for both filters.
But maybe it's easier to write it that way in Cypress?
It would probably be better to do assertion with Summarize Count, since it should return `1`, but currently returns `0`
image

**Review by nemanjaglumac — COMMENTED** [review]
> Not sure why you're creating the question twice. You can use the same source question for both filters.
> But maybe it's easier to write it that way in Cypress?

Tbh, there is no special reason - just my initial instinct. I could've gone the other way, for sure.

> It would probably be better to do assertion with Summarize Count, since it should return `1`, but currently returns `0`

Good point. Thank you.

File: frontend/test/metabase/scenarios/question/native.cy.spec.js [hunk 1]
```diff
@@ -208,4 +208,54 @@ describe("scenarios > question > native", () => {
...
+      popover().within(() => {
+        cy.findByText(filter).click();
+      });
+      cy.findByPlaceholderText("Enter some text").type("This has a value");
+      cy.findByText("Add filter").click();
+
+      cy.log("**Assert that NULL value is present**");
+      cy.findByText(`V ${filter.toLowerCase()} This has a value`);
+      cy.findByText("No results!").should("not.exist");
+      cy.findByText(/null/i);
```
[the remark this answers is not in the document]
Comment (flamber) [reply frontend/test/metabase/scenarios/question/native.cy.spec.js:258]: Yes, there's 2 rows - 1 with a string and 1 with `null` - so the "Is not" and "Does not contain" should return 1 result.

**Comment by nemanjaglumac** [comment]
@flamber - I have updated the test according to your suggestions. Thank you.

One you give it a final review and approval, I'll amend and remove `[ci skip]` and let it run in the CI.

**Review by flamber — APPROVED** [review]
Go go unskippy

**Comment by sbelak** [comment]
This repro actually finds a different bug -- the FE gives you "no results" if the dataset returned is `[[null]]`.

**Pull request merged**